mission_control/navigator_missions/nav_missions/start_gate.py

```diff
@@ -200,15 +209,18 @@ def get_size_and_build_walls(self):
         left_cone_point = self.left_f + rot_buffer
         right_cone_point = self.right_f + rot_buffer
 
-        self.walls = [self.left_b, self.left_f, left_cone_point, right_cone_point, self.right_f, self.right_b]
-
-
-        # endpoint_left_b = self.left_b + rot_buffer
-        # endpoint_right_b = self.right_b - rot_buffer
+        # Define bounds for the grid
+        self.x_lower = min(left_cone_point[0], right_cone_point[0],
```

Comment:
The cone points are always further "back" from the position of the boat so those are better to use for extreme bounds as compared to the boat